modules/extensions-spring-support/src/main/java/org/mule/module/extension/internal/capability/xml/schema/SchemaBuilder.java

```diff
@@ -791,6 +734,43 @@ protected void defaultOperation()
         schema.getSimpleTypeOrComplexTypeOrGroup().add(complexType);
     }
 
+    private AbstractDataQualifierVisitor getDataQualifierVisitor(final ExtensionType extensionType, final ExplicitGroup all, final ParameterModel parameterModel)
```

Comment:
The  getDataQualifierVisitor () name is not very descriptive. Rename to getParameterDeclarationVisitor (or something of that sort) and have the return type been simply DataQualifierVisitor